### ISSUE TITLE
native_device_orientation 0.4.1 not able to flutter pub get

### ISSUE THREAD

**Issue by kw2019ltd** [rmtmckenzie/flutter_native_device_orientation#20]
Running "flutter pub get" in xxxxx
Invalid plugin specification native_device_orientation.
Invalid "android" plugin specification.
Process finished with exit code 1

**Comment by rdev-software** [comment]
Same here

**Comment by shangyilim** [comment]
I have the same issue too, upon checking it seems that the pubspec.yaml of the plugin in incorrect:
```
      android:
        package:
          package: com.github.rmtmckenzie.native_device_orientation
          pluginClass: NativeDeviceOrientationPlugin
```

**Comment by rmtmckenzie** [comment]
whoops my bad, fix is up in 0.4.2. Sorry about that 😬
